### ISSUE TITLE
'amplify init' uses wrong profile for s3 upload for secondary environments

### ISSUE THREAD

**Issue by nosnilmot** [aws-amplify/amplify-cli#7815]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

v16.5.0

### Amplify CLI Version

5.1.2

### What operating system are you using?

MacOS Big Sur 11.4

### Amplify Categories

Not applicable

### Amplify Commands

env, init

### Describe the bug

`amplify init` uses wrong AWS Profile when uploading files to s3 when initializing a secondary environment, resulting in 'Access Denied' error message without explanation if the AWS Profile for the new environment refers to a different account (or original profile does not have permission for the new bucket).

Additionally `amplify env remove` is unable to remove environments if the upload to S3 bucket during initialization had failed.

### Expected behavior

`amplify init` should successfully initialize multiple environments when using different profiles for different accounts.

`amplify env remove` should be able to remove environments where upload to S3 failed but otherwise resources were created

### Reproduction steps

1. Initialize environment one 'envone' using AWS profile 'profile1'
 -> Amplify App is created in AWS Account belonging to profile1
 -> Configuration resources are uploaded to S3 bucket
2. Initialize environment two 'envtwo' using AWS profile 'profile2'
 -> Amplify App is created in AWS account belonging to profile2
 -> Configuration resources fail to upload to S3 bucket, environment is left in corrupt/incomplete state

Detailed steps to reproduce:

1. Before starting, to confirm no cached profile names:

```
$ cat amplify/.config/local-aws-info.json
{}
```

2. Initialize first environment 'envone' using profile 'profile1':

```
$ amplify init
Note: It is recommended to run this command from the root of your app directory
? Do you want to use an existing environment? No
? Enter a name for the environment envone
Using default provider  awscloudformation
? Select the authentication method you want to use: AWS profile

For more information on AWS Profiles, see:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html

? Please choose the profile you want to use profile1
Adding backend environment envone to AWS Amplify Console app: d9i7hb9n0syk1
⠇ Initializing project in the cloud...

CREATE_IN_PROGRESS amplify-appname-envone-151929 AWS::CloudFormation::Stack Tue Jul 27 2021 15:19:33 GMT+0100 (British Summer Time) User Initiated
CREATE_IN_PROGRESS DeploymentBucket                 AWS::S3::Bucket            Tue Jul 27 2021 15:19:36 GMT+0100 (British Summer Time)
CREATE_IN_PROGRESS AuthRole                         AWS::IAM::Role             Tue Jul 27 2021 15:19:36 GMT+0100 (British Summer Time)
CREATE_IN_PROGRESS UnauthRole                       AWS::IAM::Role             Tue Jul 27 2021 15:19:36 GMT+0100 (British Summer Time)
CREATE_IN_PROGRESS AuthRole                         AWS::IAM::Role             Tue Jul 27 2021 15:19:37 GMT+0100 (British Summer Time) Resource creation Initiated
CREATE_IN_PROGRESS UnauthRole                       AWS::IAM::Role             Tue Jul 27 2021 15:19:37 GMT+0100 (British Summer Time) Resource creation Initiated
CREATE_IN_PROGRESS DeploymentBucket                 AWS::S3::Bucket            Tue Jul 27 2021 15:19:37 GMT+0100 (British Summer Time) Resource creation Initiated
⠏ Initializing project in the cloud...

CREATE_COMPLETE UnauthRole AWS::IAM::Role Tue Jul 27 2021 15:19:53 GMT+0100 (British Summer Time)
CREATE_COMPLETE AuthRole   AWS::IAM::Role Tue Jul 27 2021 15:19:53 GMT+0100 (British Summer Time)
⠋ Initializing project in the cloud...

CREATE_COMPLETE DeploymentBucket AWS::S3::Bucket Tue Jul 27 2021 15:19:59 GMT+0100 (British Summer Time)
⠏ Initializing project in the cloud...

CREATE_COMPLETE amplify-appname-envone-151929 AWS::CloudFormation::Stack Tue Jul 27 2021 15:20:00 GMT+0100 (British Summer Time)
✔ Successfully created initial AWS cloud resources for deployments.
✔ Initialized provider successfully.
Initialized your environment successfully.

Your project has been successfully initialized and connected to the cloud!

Some next steps:
"amplify status" will show you what you've added already and if it's locally configured or deployed
"amplify add <category>" will allow you to add features like user login or a backend API
"amplify push" will build all your local backend resources and provision it in the cloud
"amplify console" to open the Amplify Console and view your project status
"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud

Pro tip:
Try "amplify add api" to create a backend API and then "amplify publish" to deploy everything
```

3. Verify correct profile name was saved for this environment

```
$ cat amplify/.config/local-aws-info.json
{
  "envone": {
    "configLevel": "project",
    "useProfile": true,
    "profileName": "profile1"
  }
}
```

4. Initialize second environment 'envtwo' using profile 'profile2' - resource creation succeeds, but file upload to S3 fails (because it is using wrong profile, and does not have access)

```
$ amplify init
Note: It is recommended to run this command from the root of your app directory
? Do you want to use an existing environment? No
? Enter a name for the environment envtwo
Using default provider  awscloudformation
? Select the authentication method you want to use: AWS profile

For more information on AWS Profiles, see:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html

? Please choose the profile you want to use profile2
Adding backend environment envtwo to AWS Amplify Console app: dxcd1wia5ve4r
⠦ Initializing project in the cloud...

CREATE_IN_PROGRESS amplify-appname-envtwo-152315 AWS::CloudFormation::Stack Tue Jul 27 2021 15:23:19 GMT+0100 (British Summer Time) User Initiated
CREATE_IN_PROGRESS AuthRole                         AWS::IAM::Role             Tue Jul 27 2021 15:23:22 GMT+0100 (British Summer Time)
CREATE_IN_PROGRESS DeploymentBucket                 AWS::S3::Bucket            Tue Jul 27 2021 15:23:22 GMT+0100 (British Summer Time)
CREATE_IN_PROGRESS UnauthRole                       AWS::IAM::Role             Tue Jul 27 2021 15:23:22 GMT+0100 (British Summer Time)
CREATE_IN_PROGRESS AuthRole                         AWS::IAM::Role             Tue Jul 27 2021 15:23:23 GMT+0100 (British Summer Time) Resource creation Initiated
CREATE_IN_PROGRESS UnauthRole                       AWS::IAM::Role             Tue Jul 27 2021 15:23:23 GMT+0100 (British Summer Time) Resource creation Initiated
CREATE_IN_PROGRESS DeploymentBucket                 AWS::S3::Bucket            Tue Jul 27 2021 15:23:23 GMT+0100 (British Summer Time) Resource creation Initiated
⠧ Initializing project in the cloud...

CREATE_COMPLETE AuthRole   AWS::IAM::Role Tue Jul 27 2021 15:23:38 GMT+0100 (British Summer Time)
CREATE_COMPLETE UnauthRole AWS::IAM::Role Tue Jul 27 2021 15:23:39 GMT+0100 (British Summer Time)
⠇ Initializing project in the cloud...

CREATE_COMPLETE DeploymentBucket AWS::S3::Bucket Tue Jul 27 2021 15:23:44 GMT+0100 (British Summer Time)
⠇ Initializing project in the cloud...

CREATE_COMPLETE amplify-appname-envtwo-152315 AWS::CloudFormation::Stack Tue Jul 27 2021 15:23:46 GMT+0100 (British Summer Time)
✔ Successfully created initial AWS cloud resources for deployments.
AccessDenied: Access Denied
    at Request.extractError (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/services/s3.js:714:35)
    at Request.callListeners (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:688:14)
    at Request.transition (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:690:12)
    at Request.callListeners (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
    at Request.emit (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:688:14)
    at Request.transition (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:690:12)
    at Request.callListeners (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
    at callNextListener (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:96:12)
    at IncomingMessage.onEnd (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/event_listeners.js:336:13)
    at IncomingMessage.emit (node:events:406:35)
    at IncomingMessage.emit (node:domain:470:12)
    at endReadableNT (node:internal/streams/readable:1331:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  code: 'AccessDenied',
  region: null,
  time: 2021-07-27T14:23:52.394Z,
  requestId: 'M7REJE410GPJVJGX',
  extendedRequestId: 'cYR1vnRWhF5HUyBqrbeKtPhwVzaBLgKUHbGK1vUBI0NfyqXmX+FybcbbKHilNU+Yb5ZQm+g3oZ8=',
  cfId: undefined,
  statusCode: 403,
  retryable: false,
  retryDelay: 6.220773111939559
}

```

5. Observe  wrong profile name is saved in `local-aws-info.json` for second environment - it has **profile1** instead of profile2, and ALL subsequent operations will fail until this is manually fixed:

```
$ cat amplify/.config/local-aws-info.json
{
  "envone": {
    "configLevel": "project",
    "useProfile": true,
    "profileName": "profile1"
  },
  "envtwo": {
    "configLevel": "project",
    "useProfile": true,
    "profileName": "profile1"
  }
}

```

This uncovers what should probably be filed as a separate issue: these failed-to-initialize environments are not removable using CLI - it aborts because file(s) are missing from S3:

```
$ amplify env remove envtwo
? Are you sure you want to continue? This CANNOT be undone. (This will delete 'envtwo' environment of the project from the cloud) Yes
⠋ Deleting resources from the cloud. This may take a few minutes...
Deleting env: envtwo.
⠸ Deleting resources from the cloud. This may take a few minutes...
Error occurred while deleting env: envtwo.
The specified key does not exist.
✖ remove env failed: The specified key does not exist.
The specified key does not exist.
NoSuchKey: The specified key does not exist.
    at Request.extractError (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/services/s3.js:714:35)
    at Request.callListeners (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
```

After manually deleting S3 bucket, remove can clean up most of the rest (ie. the above should be a non-fatal error):

```
$ amplify env remove envtwo
? Are you sure you want to continue? This CANNOT be undone. (This will delete 'envtwo' environment of the project from the cloud) Yes
⠋ Deleting resources from the cloud. This may take a few minutes...
Deleting env: envtwo.
⠋ Deleting resources from the cloud. This may take a few minutes...Unable to remove env: envtwo because deployment bucket amplify-appname-tmpdev-152315-deployment does not exist or has been deleted.
✔ Successfully removed environment from the cloud
Successfully removed environment from your project locally
```

### Log output

<details>

```
2021-07-27T14:19:29.909Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:19:29.912Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:19:29.912Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:19:29.934Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:19:29.935Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:19:29.935Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:19:29.950Z|info : amplify-provider-awscloudformation.amplify-service-permission-check.checkAmplifyServiceIAMPermission.amplifyClient.listApps([])
2021-07-27T14:19:30.380Z|info : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.createApp([{"name":"[***]name","environmentVariables":{"_LIVE_PACKAGE_UPDATES":"[{\"pkg\":\"@aws-amplify/cli\",\"type\":\"npm\",\"version\":\"latest\"}]"}}])
2021-07-27T14:19:32.010Z|info : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.getBackendEnvironment([{"appId":"[***]n0syk1","environmentName":"[***]one"}])
2021-07-27T14:19:32.623Z|error : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.getBackendEnvironment([{"appId":"[***]n0syk1","environmentName":"[***]one"}])
NotFoundException: BackendEnvironment envone does not exist.
2021-07-27T14:19:32.624Z|info : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.getBackendEnvironment([{"appId":"[***]n0syk1","environmentName":"[***]one","stackName":"[***]ify-[***]name-[***]one-[***]929","deploymentArtifacts":"amplify-appname-envone-151929-deployment"}])
2021-07-27T14:19:33.228Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:19:33.229Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:19:33.229Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:19:33.240Z|info : amplify-provider-awscloudformation.aws-cfn.cfnModel.createStack([{"StackName":"[***]ify-[***]name-[***]one-[***]929","Capabilities":["CAPABILITY_NAMED_IAM","CAPABILITY_AUTO_EXPAND"],"TemplateBody":"{\"AWSTemplateFormatVersion\":\"2010-09-09\",\"Description\":\"Root Stack for AWS Amplify CLI\",\"Parameters\":{\"[***]cketName\":{\"Description\":\"Name of the common deployment bucket provided by the parent stack\",\"Type\":\"String\",\"Default\":\"DeploymentBucket\"},\"[***]eName\":{\"Type\":\"String\",\"Default\":\"AuthRoleName\"},\"[***]leName\":{\"Type\":\"String\",\"Default\":\"UnauthRoleName\"}},\"Resources\":{\"DeploymentBucket\":{\"Type\":\"AWS::S3::Bucket\",\"DeletionPolicy\":\"Retain\",\"Properties\":{\"BucketName\":{\"Ref\":\"DeploymentBucketName\"},\"BucketEncryption\":{\"ServerSideEncryptionConfiguration\":[{\"ServerSideEncryptionByDefault\":{\"SSEAlgorithm\":\"AES256\"}}]}}},\"AuthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"AuthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}},\"UnauthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"UnauthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}}},\"Outputs\":{\"Region\":{\"Description\":\"CloudFormation provider root stack Region\",\"Value\":{\"Ref\":\"AWS::Region\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-Region\"}}},\"StackName\":{\"Description\":\"CloudFormation provider root stack ID\",\"Value\":{\"Ref\":\"AWS::StackName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackName\"}}},\"StackId\":{\"Description\":\"CloudFormation provider root stack name\",\"Value\":{\"Ref\":\"AWS::StackId\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackId\"}}},\"DeploymentBucketName\":{\"Description\":\"CloudFormation provider root stack deployment bucket name\",\"Value\":{\"Ref\":\"DeploymentBucketName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-DeploymentBucketName\"}}},\"AuthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"AuthRole\",\"Arn\"]}},\"UnauthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"UnauthRole\",\"Arn\"]}},\"AuthRoleName\":{\"Value\":{\"Ref\":\"AuthRole\"}},\"UnauthRoleName\":{\"Value\":{\"Ref\":\"UnauthRole\"}}}}","Parameters":[{"ParameterKey":"DeploymentBucketName","ParameterValue":"amplify-appname-envone-151929-deployment"},{"ParameterKey":"AuthRoleName","ParameterValue":"amplify-appname-envone-151929-authRole"},{"ParameterKey":"UnauthRoleName","ParameterValue":"amplify-appname-envone-151929-unauthRole"}],"Tags":[{"Key":"[***]tack","Value":"dev"},{"Key":"[***]ication","Value":"appname"}]}])
2021-07-27T14:19:34.163Z|info : amplify-provider-awscloudformation.aws-cfn.cfnModel.createStack([{"StackName":"[***]ify-[***]name-[***]one-[***]929","Capabilities":["CAPABILITY_NAMED_IAM","CAPABILITY_AUTO_EXPAND"],"TemplateBody":"{\"AWSTemplateFormatVersion\":\"2010-09-09\",\"Description\":\"Root Stack for AWS Amplify CLI\",\"Parameters\":{\"[***]cketName\":{\"Description\":\"Name of the common deployment bucket provided by the parent stack\",\"Type\":\"String\",\"Default\":\"DeploymentBucket\"},\"[***]eName\":{\"Type\":\"String\",\"Default\":\"AuthRoleName\"},\"[***]leName\":{\"Type\":\"String\",\"Default\":\"UnauthRoleName\"}},\"Resources\":{\"DeploymentBucket\":{\"Type\":\"AWS::S3::Bucket\",\"DeletionPolicy\":\"Retain\",\"Properties\":{\"BucketName\":{\"Ref\":\"DeploymentBucketName\"},\"BucketEncryption\":{\"ServerSideEncryptionConfiguration\":[{\"ServerSideEncryptionByDefault\":{\"SSEAlgorithm\":\"AES256\"}}]}}},\"AuthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"AuthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}},\"UnauthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"UnauthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}}},\"Outputs\":{\"Region\":{\"Description\":\"CloudFormation provider root stack Region\",\"Value\":{\"Ref\":\"AWS::Region\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-Region\"}}},\"StackName\":{\"Description\":\"CloudFormation provider root stack ID\",\"Value\":{\"Ref\":\"AWS::StackName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackName\"}}},\"StackId\":{\"Description\":\"CloudFormation provider root stack name\",\"Value\":{\"Ref\":\"AWS::StackId\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackId\"}}},\"DeploymentBucketName\":{\"Description\":\"CloudFormation provider root stack deployment bucket name\",\"Value\":{\"Ref\":\"DeploymentBucketName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-DeploymentBucketName\"}}},\"AuthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"AuthRole\",\"Arn\"]}},\"UnauthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"UnauthRole\",\"Arn\"]}},\"AuthRoleName\":{\"Value\":{\"Ref\":\"AuthRole\"}},\"UnauthRoleName\":{\"Value\":{\"Ref\":\"UnauthRole\"}}}}","Parameters":[{"ParameterKey":"DeploymentBucketName","ParameterValue":"amplify-appname-envone-151929-deployment"},{"ParameterKey":"AuthRoleName","ParameterValue":"amplify-appname-envone-151929-authRole"},{"ParameterKey":"UnauthRoleName","ParameterValue":"amplify-appname-envone-151929-unauthRole"}],"Tags":[{"Key":"[***]tack","Value":"dev"},{"Key":"[***]ication","Value":"appname"}]}])
2021-07-27T14:19:39.180Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:19:39.731Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]42:stack/[***]ify-[***]name-[***]one-[***]929/[***]9a50-[***]e5-[***]eb-[***]5a-[***]b5dba"}])
2021-07-27T14:19:44.175Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:19:44.925Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]42:stack/[***]ify-[***]name-[***]one-[***]929/[***]9a50-[***]e5-[***]eb-[***]5a-[***]b5dba"}])
2021-07-27T14:19:49.176Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:19:49.926Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]42:stack/[***]ify-[***]name-[***]one-[***]929/[***]9a50-[***]e5-[***]eb-[***]5a-[***]b5dba"}])
2021-07-27T14:19:54.179Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:19:54.726Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]42:stack/[***]ify-[***]name-[***]one-[***]929/[***]9a50-[***]e5-[***]eb-[***]5a-[***]b5dba"}])
2021-07-27T14:19:59.179Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:19:59.735Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]42:stack/[***]ify-[***]name-[***]one-[***]929/[***]9a50-[***]e5-[***]eb-[***]5a-[***]b5dba"}])
2021-07-27T14:20:04.184Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:20:06.162Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:20:06.173Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:20:06.173Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:20:06.194Z|info : amplify-provider-awscloudformation.attach-backend.storeCurrentCloudBackend.s3.uploadFile([{"Key":"[***]rent-[***]ud-[***]d.zip"}])
2021-07-27T14:20:06.201Z|info : amplify-provider-awscloudformation.aws-s3.uploadFile.s3.putObject([{"Key":"[***]rent-[***]ud-[***]d.zip","Bucket":"[***]ify-[***]name-[***]one-[***]929-[***]ment"}])
2021-07-27T14:20:16.700Z|info : amplify-provider-awscloudformation.attach-backend.uploadFile.s3.uploadFile([{"Key":"[***]ify-[***]json"}])
2021-07-27T14:20:16.700Z|info : amplify-provider-awscloudformation.aws-s3.uploadFile.s3.putObject([{"Key":"[***]ify-[***]json","Bucket":"[***]ify-[***]name-[***]one-[***]929-[***]ment"}])
2021-07-27T14:20:17.441Z|info : amplify-provider-awscloudformation.attach-backend.uploadFile.s3.uploadFile([{"Key":"[***]end-[***].json"}])
2021-07-27T14:20:17.442Z|info : amplify-provider-awscloudformation.aws-s3.uploadFile.s3.putObject([{"Key":"[***]end-[***].json","Bucket":"[***]ify-[***]name-[***]one-[***]929-[***]ment"}])
2021-07-27T14:20:59.308Z|info : amplify init core  
2021-07-27T14:20:59.373Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2021-07-27T14:23:15.220Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile2"])
2021-07-27T14:23:15.223Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:23:15.223Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile2"])
2021-07-27T14:23:15.230Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:23:15.232Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:23:15.232Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:23:15.242Z|info : amplify-provider-awscloudformation.amplify-service-permission-check.checkAmplifyServiceIAMPermission.amplifyClient.listApps([])
2021-07-27T14:23:15.684Z|info : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.listApps([{"maxResults":25}])
2021-07-27T14:23:16.091Z|info : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.createApp([{"name":"[***]name","environmentVariables":{"_LIVE_PACKAGE_UPDATES":"[{\"pkg\":\"@aws-amplify/cli\",\"type\":\"npm\",\"version\":\"latest\"}]"}}])
2021-07-27T14:23:18.116Z|info : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.getBackendEnvironment([{"appId":"[***]a5ve4r","environmentName":"[***]two"}])
2021-07-27T14:23:18.730Z|error : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.getBackendEnvironment([{"appId":"[***]a5ve4r","environmentName":"[***]two"}])
NotFoundException: BackendEnvironment envtwo does not exist.
2021-07-27T14:23:18.731Z|info : amplify-provider-awscloudformation.amplify-service-manager.init.amplifyClient.getBackendEnvironment([{"appId":"[***]a5ve4r","environmentName":"[***]two","stackName":"[***]ify-[***]name-[***]two-[***]315","deploymentArtifacts":"amplify-appname-envtwo-152315-deployment"}])
2021-07-27T14:23:19.055Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:23:19.056Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:23:19.056Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:23:19.069Z|info : amplify-provider-awscloudformation.aws-cfn.cfnModel.createStack([{"StackName":"[***]ify-[***]name-[***]two-[***]315","Capabilities":["CAPABILITY_NAMED_IAM","CAPABILITY_AUTO_EXPAND"],"TemplateBody":"{\"AWSTemplateFormatVersion\":\"2010-09-09\",\"Description\":\"Root Stack for AWS Amplify CLI\",\"Parameters\":{\"[***]cketName\":{\"Description\":\"Name of the common deployment bucket provided by the parent stack\",\"Type\":\"String\",\"Default\":\"DeploymentBucket\"},\"[***]eName\":{\"Type\":\"String\",\"Default\":\"AuthRoleName\"},\"[***]leName\":{\"Type\":\"String\",\"Default\":\"UnauthRoleName\"}},\"Resources\":{\"DeploymentBucket\":{\"Type\":\"AWS::S3::Bucket\",\"DeletionPolicy\":\"Retain\",\"Properties\":{\"BucketName\":{\"Ref\":\"DeploymentBucketName\"},\"BucketEncryption\":{\"ServerSideEncryptionConfiguration\":[{\"ServerSideEncryptionByDefault\":{\"SSEAlgorithm\":\"AES256\"}}]}}},\"AuthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"AuthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}},\"UnauthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"UnauthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}}},\"Outputs\":{\"Region\":{\"Description\":\"CloudFormation provider root stack Region\",\"Value\":{\"Ref\":\"AWS::Region\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-Region\"}}},\"StackName\":{\"Description\":\"CloudFormation provider root stack ID\",\"Value\":{\"Ref\":\"AWS::StackName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackName\"}}},\"StackId\":{\"Description\":\"CloudFormation provider root stack name\",\"Value\":{\"Ref\":\"AWS::StackId\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackId\"}}},\"DeploymentBucketName\":{\"Description\":\"CloudFormation provider root stack deployment bucket name\",\"Value\":{\"Ref\":\"DeploymentBucketName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-DeploymentBucketName\"}}},\"AuthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"AuthRole\",\"Arn\"]}},\"UnauthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"UnauthRole\",\"Arn\"]}},\"AuthRoleName\":{\"Value\":{\"Ref\":\"AuthRole\"}},\"UnauthRoleName\":{\"Value\":{\"Ref\":\"UnauthRole\"}}}}","Parameters":[{"ParameterKey":"DeploymentBucketName","ParameterValue":"amplify-appname-envtwo-152315-deployment"},{"ParameterKey":"AuthRoleName","ParameterValue":"amplify-appname-envtwo-152315-authRole"},{"ParameterKey":"UnauthRoleName","ParameterValue":"amplify-appname-envtwo-152315-unauthRole"}],"Tags":[{"Key":"[***]tack","Value":"envone"},{"Key":"[***]ication","Value":"appname"}]}])
2021-07-27T14:23:19.893Z|info : amplify-provider-awscloudformation.aws-cfn.cfnModel.createStack([{"StackName":"[***]ify-[***]name-[***]two-[***]315","Capabilities":["CAPABILITY_NAMED_IAM","CAPABILITY_AUTO_EXPAND"],"TemplateBody":"{\"AWSTemplateFormatVersion\":\"2010-09-09\",\"Description\":\"Root Stack for AWS Amplify CLI\",\"Parameters\":{\"[***]cketName\":{\"Description\":\"Name of the common deployment bucket provided by the parent stack\",\"Type\":\"String\",\"Default\":\"DeploymentBucket\"},\"[***]eName\":{\"Type\":\"String\",\"Default\":\"AuthRoleName\"},\"[***]leName\":{\"Type\":\"String\",\"Default\":\"UnauthRoleName\"}},\"Resources\":{\"DeploymentBucket\":{\"Type\":\"AWS::S3::Bucket\",\"DeletionPolicy\":\"Retain\",\"Properties\":{\"BucketName\":{\"Ref\":\"DeploymentBucketName\"},\"BucketEncryption\":{\"ServerSideEncryptionConfiguration\":[{\"ServerSideEncryptionByDefault\":{\"SSEAlgorithm\":\"AES256\"}}]}}},\"AuthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"AuthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}},\"UnauthRole\":{\"Type\":\"AWS::IAM::Role\",\"Properties\":{\"RoleName\":{\"Ref\":\"UnauthRoleName\"},\"AssumeRolePolicyDocument\":{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"\",\"Effect\":\"Deny\",\"Principal\":{\"Federated\":\"cognito-identity.amazonaws.com\"},\"Action\":\"sts:AssumeRoleWithWebIdentity\"}]}}}},\"Outputs\":{\"Region\":{\"Description\":\"CloudFormation provider root stack Region\",\"Value\":{\"Ref\":\"AWS::Region\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-Region\"}}},\"StackName\":{\"Description\":\"CloudFormation provider root stack ID\",\"Value\":{\"Ref\":\"AWS::StackName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackName\"}}},\"StackId\":{\"Description\":\"CloudFormation provider root stack name\",\"Value\":{\"Ref\":\"AWS::StackId\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-StackId\"}}},\"DeploymentBucketName\":{\"Description\":\"CloudFormation provider root stack deployment bucket name\",\"Value\":{\"Ref\":\"DeploymentBucketName\"},\"Export\":{\"Name\":{\"Fn::Sub\":\"${AWS::StackName}-DeploymentBucketName\"}}},\"AuthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"AuthRole\",\"Arn\"]}},\"UnauthRoleArn\":{\"Value\":{\"Fn::GetAtt\":[\"UnauthRole\",\"Arn\"]}},\"AuthRoleName\":{\"Value\":{\"Ref\":\"AuthRole\"}},\"UnauthRoleName\":{\"Value\":{\"Ref\":\"UnauthRole\"}}}}","Parameters":[{"ParameterKey":"DeploymentBucketName","ParameterValue":"amplify-appname-envtwo-152315-deployment"},{"ParameterKey":"AuthRoleName","ParameterValue":"amplify-appname-envtwo-152315-authRole"},{"ParameterKey":"UnauthRoleName","ParameterValue":"amplify-appname-envtwo-152315-unauthRole"}],"Tags":[{"Key":"[***]tack","Value":"envone"},{"Key":"[***]ication","Value":"appname"}]}])
2021-07-27T14:23:24.902Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]two-[***]315"}])
2021-07-27T14:23:25.443Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]04:stack/[***]ify-[***]name-[***]two-[***]315/[***]d0a0-[***]e6-[***]eb-[***]f3-[***]8111a"}])
2021-07-27T14:23:29.903Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]two-[***]315"}])
2021-07-27T14:23:30.434Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]04:stack/[***]ify-[***]name-[***]two-[***]315/[***]d0a0-[***]e6-[***]eb-[***]f3-[***]8111a"}])
2021-07-27T14:23:34.906Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]two-[***]315"}])
2021-07-27T14:23:35.448Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]04:stack/[***]ify-[***]name-[***]two-[***]315/[***]d0a0-[***]e6-[***]eb-[***]f3-[***]8111a"}])
2021-07-27T14:23:39.907Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]two-[***]315"}])
2021-07-27T14:23:40.441Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]04:stack/[***]ify-[***]name-[***]two-[***]315/[***]d0a0-[***]e6-[***]eb-[***]f3-[***]8111a"}])
2021-07-27T14:23:44.909Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]two-[***]315"}])
2021-07-27T14:23:45.475Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]rmation:us-[***]st-[***]04:stack/[***]ify-[***]name-[***]two-[***]315/[***]d0a0-[***]e6-[***]eb-[***]f3-[***]8111a"}])
2021-07-27T14:23:49.909Z|info : amplify-provider-awscloudformation.aws-cfn.getStackEvents.cfnModel.describeStackEvents([{"StackName":"[***]ify-[***]name-[***]two-[***]315"}])
2021-07-27T14:23:51.824Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:23:51.836Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:23:51.836Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:23:51.860Z|info : amplify-provider-awscloudformation.attach-backend.storeCurrentCloudBackend.s3.uploadFile([{"Key":"[***]rent-[***]ud-[***]d.zip"}])
2021-07-27T14:23:51.866Z|info : amplify-provider-awscloudformation.aws-s3.uploadFile.s3.putObject([{"Key":"[***]rent-[***]ud-[***]d.zip","Bucket":"[***]ify-[***]name-[***]two-[***]315-[***]ment"}])
2021-07-27T14:23:52.395Z|error : amplify-provider-awscloudformation.aws-s3.uploadFile.s3([{"Key":"[***]rent-[***]ud-[***]d.zip","Bucket":"[***]ify-[***]name-[***]two-[***]315-[***]ment"}])
AccessDenied: Access Denied



2021-07-27T14:40:23.556Z|info : amplify env core checkout envone
2021-07-27T14:40:35.684Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:40:35.686Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:40:35.686Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:40:35.705Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["profile1"])
2021-07-27T14:40:35.705Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"cli_pager":"","output":"json","region":"us-east-2"}])
2021-07-27T14:40:35.705Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["profile1"])
2021-07-27T14:40:35.715Z|info : amplify-provider-awscloudformation.zip-util.downloadZip.s3.getFile([{"Key":"[***]rent-[***]ud-[***]d.zip"},null])
2021-07-27T14:40:35.717Z|info : amplify-provider-awscloudformation.aws-s3.s3.getFile([{"Key":"[***]rent-[***]ud-[***]d.zip","Bucket":"[***]ify-[***]name-[***]one-[***]929-[***]ment"}])
2021-07-27T14:40:38.706Z|info : amplify-provider-awscloudformation.initialize-env.run.cfn.updateamplifyMetaFileWithStackOutputs([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:40:38.708Z|info : amplify-provider-awscloudformation.aws-cfn.updateamplifyMetaFileWithStackOutputs.cfn.describeStackResources([{"StackName":"[***]ify-[***]name-[***]one-[***]929"}])
2021-07-27T14:40:39.302Z|info : amplify-provider-awscloudformation.aws-s3.s3.getFile([{"Key":"[***]ify-[***]json","Bucket":"[***]ify-[***]name-[***]one-[***]929-[***]ment"}])
```

</details>

**Comment by meekgeek** [comment]
I had the same issue, the workaround is to temporarily change your first profile to the one you're trying to use for the second. :)

**Comment by djsmedes** [comment]
Same issue. We just upgraded from amplify CLI version 4.41.2 where this issue was not present, so it's been introduced sometime since then.

**Comment by nosnilmot** [comment]
This issue was introduced between Amplify CLI version 5.1.1 and 5.1.2

These versions have been tested and confirmed to work: 4.41.2, 4.52.0, 5.0.0, 5.1.0, 5.1.1
These versions have been confirmed to fail: 5.1.2, 5.2.0, git HEAD

Using 'git bisect' I have narrowed the problem down to this commit:
7471c5fcc86af0e17a967066a388f67891f93355 - fix: #7441 - init from git prompts for credentials twice (#7682)

I cannot see what specific part of that commit is problematic, unfortunately there is a lot of noise in there from the renaming of `awsConfig` to `awsConfigInfo`.

**Comment by kaustavghosh06** [comment]
We have a fix for this in the pipeline and we're working on a release for this - https://github.com/aws-amplify/amplify-cli/pull/7990

**Comment by kaustavghosh06** [comment]
Closing as a duplicate of  #7977
